docs/user/estimate_migration.rst

```diff
@@ -0,0 +1,160 @@
+====================
+Migration Estimation
+====================
+
```

Comment:
By my opinion we should add some overview here - what is the goal of this CF part and how we can use execution results of discovery module. Because later we have a subsection named `Object Selection for Migration` - it can  be confusing to someone who doesn't know about CF and read the documentation in the first time, because we have similar filters in `configs/filter.yaml` described in `docs/user/configuration.rst`.